core/kazoo_csv/src/kz_csv.erl

```diff
@@ -225,10 +304,11 @@ map_io_indices(Header, CSVHeader) ->
     lists:foldl(MapF, #{}, IndexToCSVHeader).
 
 %% @private
--spec cell_to_binary(cell()) -> csv().
+-spec cell_to_binary(cell()) -> binary().
```

Comment:
A cell != a csv right? So why would the return type be `csv()` (ignoring that `csv()` is equivalent to `binary()`). Seems misleading to say that converting a cell to binary returns a csv.